src/mudlet.cpp

```diff
@@ -751,6 +751,10 @@ void mudlet::slot_close_profile_requested( int tab )
     else
         pH->mpConsole->mUserAgreedToCloseConsole = true;
     pH->closingDown();
+
+    // disconnect telnet early.
```

Comment:
Mind adding a comment explaining the _why_? What it does it the obvious part, why not so :wink: